docs/monoDrive_home/Weather.md

```diff
@@ -71,139 +54,167 @@ window for the Actor. Scroll down to the "Weather Description" in the "Details"
 window to find the weather settings pictured below.
 
 <p class="img_container">
-  <img class="lg_img" src="../img/details_weather_actor.png" />
+  <img class="lg_img" src="../img/weather_ui.png" />
 </p>
 
-* **Load File Button:** Loads the current weather profile from the Profile Name 
-in file specified in the "File Name" field.
-* **Save File Button:** Saves the current weather profile to the file specified 
-in the "File Name" field. If the profile already existed in the file, it will be 
-updated with the current values. When saved, both this `.ini` file and a `.json` 
-file will be saved for use with the simulator and client.
-* **Configuration File Name:** The name of the `.ini` file to save the current
-profile to. 
-* **Profile Name:** Located in the Name field under the Weather settings,
-specifies the human readable name of the weather profile to save to.
-
-<p>&nbsp;</p>
+**Import Weather Button:** Opens a file dialog to import a weather `.json` file. This will populate the Weather Description list with the new descriptions while removing the currently loaded descriptions.  
+**Export Weather Button:** Saves the weather description list to the `.json` file listed. This allows you to use the weather descriptions from the client. If no file path is listed in Weather File Path then you will receive a file dialogue to select or create the file.  
+**Export Weather As Button:** Opens a file dialogue to select or create a new weather `.json` file for export. 
+**Mark Actors to Save:** The weather actor works by modifying the other actors responsible for producing weather affects such as the fog actor, directional light component, sky light etc. When this box is selected, modifying the current weather description or changing the selected weather description will mark the affected actors as dirty. This will allow you to save the level the actor belongs to so that whenever you load the map, weather will be set just as you see it right now.  
+**Artificial Lighting Actor Classes:** If you have added custom actors to the map which should enable or disable some lighting you can add them to this list to receive the notification that the map is now in night settings so that they will enable/disable automatically.  
```

Comment:
removing you, some formatting changes. Adding custom actors to the map should enable or disable some lighting. This configuration enables users to add the custom actors to this list to receive the notification that the map is now in night settings, so that the custom actors will enable/disable automatically.

Reply:
fixed, mostly? i don't know, should just make suggestion for acceptance.